src/Writers/HTMLWriter.jl

```diff
@@ -1030,7 +1030,11 @@ mdconvert(h::Markdown.Header{N}, parent; kwargs...) where {N} = DOM.Tag(Symbol("
 
 mdconvert(::Markdown.HorizontalRule, parent; kwargs...) = Tag(:hr)()
 
-mdconvert(i::Markdown.Image, parent; kwargs...) = Tag(:img)[:src => i.url, :alt => i.alt]
+mdconvert(i::Markdown.Image, parent; kwargs...) = if endswith(i.url, ".webm")
+    Tag(:video)[:src => i.url, :width => "800", :controls => "true", Symbol("data-title") => i.alt]
```

Comment:
Unless I am missing something, would it not be better to style the `<video>` tag with CSS, rather than using the `width` attribute? I think we just should apply the same style as for `img`, i.e. just set `max-width`:

https://github.com/JuliaDocs/Documenter.jl/blob/6e35c14093577a23a78952e097b476bd95884543/assets/html/documenter.css#L82-L84